packages/gulp-conventional-changelog/index.js

```diff
@@ -1,15 +1,14 @@
 'use strict'
 
 const addStream = require('add-stream')
-const assign = require('object-assign')
 const concat = require('concat-stream')
 const conventionalChangelog = require('conventional-changelog')
 const PluginError = require('plugin-error')
 const fancyLog = require('fancy-log')
 const through = require('through2')
 
 module.exports = function (opts, context, gitRawCommitsOpts, parserOpts, writerOpts) {
-  opts = assign({
+  opts = Object.assign({
```

Comment:
How about just using object destructing instead. vscode have better understanding of it's properties that way...

```js
opts = {
    // TODO: remove this when gulp get's a real logger with levels
    verbose: process.argv.indexOf('--verbose') !== -1,
    ...opts
}
```

Reply:
i do agree that `object destructing` has a better syntax, but sadly we are not able to use to as this code has to work on node 10